### ISSUE TITLE
[2106.11189] Regularization is all you Need: Simple Neural Nets can Excel on Tabular Data

### ISSUE THREAD

**Issue by riron1206** [riron1206/read_paper#96]
## 概要
https://arxiv.org/abs/2106.11189
DLモデル のパラメータチューニングを頑張ったらXGBoostを上回ったと主張し、
テーブルデータのDeepモデルを改善する鍵は、data augmentation、skip connection、モデルアンサンブル（dropout, snapshot ensembles など）などの正則化を使うことと考えて試した論文

実験条件ありすぎ
- 分類のみ。回帰は実験してない
- バランスのとれたデータセットのみを使用（少数派クラスと多数派クラスの比率が0.05以上）
- データ数（行数）がある程度あるデータセットのみ使用（最小のデータセットは452行、最大のデータセットは416188行）
- 外れ値を持つ、欠損ラベルを持つ、半教師データ、ストリーミングデータ（継続的に生成され続けるデータ. システムログなど）のデータセットは検証してない
- 注目すべき重要な点は、最近のニューラルネットワークアーキテクチャ（セクション5.4）も我々の正則化カクテルから恩恵を受けることができるが、ベースラインライブラリに正則化を統合するには、かなりのコーディング作業が必要となることである。

**意地でもニューラルネットはGBDTに勝てると言いたいみたいだが、XGBoostとの比較が不公平らしい**
https://twitter.com/giffmana/status/1407221895649542147

## 結果
正則化を最適化したらテーブルデータのDeepモデルがGBDTより高精度になることを示した
ニューラルネットワークがXGBoostを大幅に上回った
正則化された単純な深いMLPがXGBoostを上回る

ディープニューラルネットワークのフィッティングには時間がかかり、また、我々の正則化カクテルにはデータセットごとのハイパーパラメータの最適化が必要

## コメント
